### DISCUSSION TITLE
A question about CV for 4th IFC

### DISCUSSION THREAD

**Discussion by gangliu-github** [ttadano/alamode#196] [Q&A] [answered]
I generated independent random sampling configurations from the harmonic PES at 300 K, following the 7.6 of the Tutorial. I want to obtain the 4th IFCs. But when I performed the CV, I found the RESIDUAL error was above 50%. I also found the strange information:
  RESIDUAL (%): 68.1987
  Number of non-zero  HARMONIC FCs : 0
  Number of non-zero   ANHARM3 FCs : 0
  Number of non-zero   ANHARM4 FCs : 1
What is the problem? The files are uploaded.
[cv.zip](https://github.com/ttadano/alamode/files/15144461/cv.zip)

**Comment by ttadano** [comment, marked as answer]
In `cv.log`, you can see that the L1 penalty `alpha` value is very large for the 2dn and 4th CV sets, while they are small for the 1st and 3rd CV sets. This implies that your training data contains some anomalies (e.g., unconverged results or unphysically high-energy structures).

**Reply by gangliu-github** [reply]
Thanks for your reply! Now I use the random structures from AIMD calculations, but the same problem also appears. Then, how to solve this problem? If I re-generate these random structures, may the problem be solved?
